### ISSUE TITLE
Nub all list values values?

### ISSUE THREAD

**Issue by garyb** [slamdata/purescript-markdown#44]
Similar to #43, it might be better if we take care of de-duplicating values in the markdown itself, as having duplicate checkboxes, radio buttons or dropdown elements makes little sense and may even cause issues.

**Comment by jonsterling** [comment]
I would be in favor of this unless someone can find a problem with the idea... I got very confused this morning as a result of the current behavior.

**Comment by garyb** [comment]
I don't think there's a technical reason why we can't, but I would say a query populating one of these lists "should" use `DISTINCT` perhaps, in our case, so I wasn't sure whether we wanted to do it in the front end or not. 

The only downside I can see is that doing so may hide poorly performing queries (a result set with 100,000 rows but only 2 unique values, that kind of thing).

**Comment by jonsterling** [comment]
Hmm, that's a good point about `DISTINCT`.

**Comment by jdegoes** [comment]
Just on principle, I'd say it's best to avoid duplicating functionality (although the point about the elements requiring uniqueness is valid and worth consideration). On more pragmatic grounds, you'd have to implement an `Eq` for the Quasar's data model, which is not that simple to do, nor spec'd out because it's currently connector-dependent.

**Comment by jonsterling** [comment]
@jdegoes Good point; regarding `Eq`, at this level, we just have strings in fact, so we can definitely compare them—but it may not be a good idea to do so :wink:
